### PR TITLE
A07 scheduler improvement, colors fixed

### PR DESCRIPTION
**Pull request by jsflca2012**

### CODE REVIEW

File: src/components/BackButton.js [hunk 1]
```diff
@@ -16,6 +18,17 @@ class BackButton extends Component {
...
 
+	label() {
+		var label = "";
+		if (this.props.label == null) {
+			label = "Back";
+		}
+		if (this.props.label == "cancel") {
+			label = "Cancel";
+		}
+		return label;
```
Comment (NeilAllavarpu) [inline src/components/BackButton.js:29]: Rewrite as `return label || "Back";`

File: src/components/FeatureViews/Scheduler/EditSessionView.js [hunk 2]
```diff
@@ -33,6 +37,9 @@ const EditSessionView = ({course, session, editSelection}) => {
 		[dispatch]
 	);
 	const history = useHistory();
+	const location = useLocation();
+	const { course, session } = location.state;
```
Comment (NeilAllavarpu) [inline src/components/FeatureViews/Scheduler/EditSessionView.js:41]: You can't use `location.state` to get this - it won't load the course/session if the user tries to open this link without pressing a certain button.

File: src/components/FeatureViews/Scheduler/EditSessionView.js [hunk 3]
```diff
@@ -33,6 +37,9 @@ const EditSessionView = ({course, session, editSelection}) => {
...
+	console.log(course, session);
```
Comment (NeilAllavarpu) [inline src/components/FeatureViews/Scheduler/EditSessionView.js:42]: Remove console.log

File: src/components/FeatureViews/Scheduler/EditSessionView.js (hunk 2 above)
Comment (dhuangg) [reply src/components/FeatureViews/Scheduler/EditSessionView.js:41]: @NeilAllavarpu we're going to open a new PR for this - @jsflca2012 will be migrating session view to graphQL

**Pull request merged**